src/main/java/ca/uhn/fhir/jpa/starter/JpaRestfulServer.java

```diff
@@ -45,4 +48,32 @@ private void registerLocationHierarchyTypes() {
 	  getFhirContext().registerCustomType(ChildTreeNode.class);
   }
 
+
+	private void registerPracitionerDetailsTypes() {
+		IFhirResourceDao<Practitioner> practitionerIFhirResourceDao = daoRegistry.getResourceDao("Practitioner");
+		IFhirResourceDao<PractitionerRole> practitionerRoleIFhirResourceDao = daoRegistry.getResourceDao("PractitionerRole");
+		IFhirResourceDao<CareTeam> careTeamIFhirResourceDao = daoRegistry.getResourceDao("CareTeam");
+		IFhirResourceDao<OrganizationAffiliation> organizationAffiliationIFhirResourceDao = daoRegistry.getResourceDao("OrganizationAffiliation");
+		IFhirResourceDao<Organization> organizationIFhirResourceDao = daoRegistry.getResourceDao("Organization");
+		IFhirResourceDao<Location> locationIFhirResourceDao = daoRegistry.getResourceDao(LOCATION);
```

Comment:
Please move  the strings to a constants file

Reply:
Will be covered under the scope of https://github.com/opensrp/hapi-fhir-jpaserver-starter/issues/25